### PR TITLE
"To be reborn one must die, Tenar."

### PR DESCRIPTION
**Pull request by saengowp**
This PR fix the issue in which the backend crashed but the container continues to run, preventing container from being restart automatically.

Ideally the code should not crash, but, as we have witnessed on yesterday's practice demo, it DO crash in the worse way possible.

### CODE REVIEW

File: start_production_docker.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+NODE_ENV=production node dist/apps/doji-backend/main.js &
+cd dist/apps/doji-frontend/
+../../../node_modules/.bin/next start -p 3000 &
+echo "Started Doji Backend and Frontend"
+cd ../../../
+
+nginx
```
Comment (saenyakorn) [inline start_production_docker.sh:9]: should this be `nginx -g "deamon off;"`

File: start_production_docker.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+wait -n
+
+echo "Doji crash detected, exiting"
+
+exit 1
```
Comment (saenyakorn) [inline start_production_docker.sh:15]:
```suggestion
exit 1

```

**Review by saenyakorn — APPROVED** [review]

File: start_production_docker.sh (hunk 1 above)
Comment (saengowp) [reply start_production_docker.sh:9]: No it should not. We want to daemonize it.

Comment (saenyakorn) [reply start_production_docker.sh:9]: ok

**Pull request merged**